docs/libpod.conf.5.md

```diff
@@ -16,10 +16,10 @@ libpod to manage containers.
   Default OCI runtime to use if nothing is specified in **runtimes**
 
 **runtimes**
-  For each OCI runtime, specify a list of paths to look for.  The first one found is used.
+  For each OCI runtime, specify a list of paths to look for.  The first one found is used. If the paths are empty or no valid path was found, then the `$PATH` environment variable will be looked up as fallback.
```

Comment:
s/looked up as fallback/used as the fallback/   
Ditto other spots.

Reply:
Alright, changed as suggested :+1: